src/test/java/TrainingTimeManagerTest.java

```diff
@@ -0,0 +1,92 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalDateTime;
+
+public class TrainingTimeManagerTest {
```

Comment:
please check if it would be possible to rework tests that are similar and have only different test data using parametrized tests